Operational-Procedures/Main-Telescope/ComCam-Master-Bias-Procedure/comcam-master-bias-procedure.rst

```diff
@@ -0,0 +1,110 @@
+.. |author| replace:: *Andrés A. Plazas Malagón*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _ComCam-Master-Bias-Procedure:
+
+###########################
+ComCam Master Bias Generation Procedure
+###########################
+
+.. _ComCam-Master-Bias-Procedure-Overview:
+
+Overview
+========
+
+This procedure describes how to call scripts to produce a master bias for LSSTComCam from the LSST Operations and Visualization Enviroment (LOVE) at the Summit :ref:`operational environment <Observing-Interface-Operational-Environments>`. 
+
+The script will: 
+
+- command LSSTComCam to take a number of bias frames,
+- call the Rubin Science Pipelines bias generation pipetask via the OCS-Controlled Pipeline System (OCPS),
+- verify the resulting bias (see the package `cp_verify`_ and `DMTN-101`_)
+- certify the resulting bias with a given range of validity dates.
+  
+For more information about calibrations production (including verification and certification), please consult the `Constructing Calibrations documentation`_.
+
+.. _cp_verify: https://github.com/lsst/cp_verify
+.. _DMTN-101: https://dmtn-101.lsst.io/
+.. _Constructing Calibrations documentation: https://lsst.ncsa.illinois.edu/~czw/pipelines_lsst_io/_build/html/modules/lsst.cp.pipe/constructing-calibrations.html 
+
+.. _ComCam-Master-Bias-Procedure-Prerequisites:
+
+Prerequisites
+=============
+
+- You should be logged into the LSST Operations and Visualization Enviroment (LOVE) at the Summit :ref:`operational environment <Observing-Interface-Operational-Environments>`.
+- The script assumes  that the LSSTComcam and OCPS are both ``ENABLED``, and that latter has been ``ENABLED`` with the configuration of ``LSSTComCam``.
+- A `butler`_ is needed to run the pipetask bias command.
+
+.. _butler: https://pipelines.lsst.io/v/daily/modules/lsst.daf.butler/index.html
+
+.. _ComCam-Master-Bias-Procedure-Post-Conditions:
+
+Post-Condition
+==============
+
+- A master bias image per detector will be certified in an individual butler CALIBRATION `collection`_ that can be grouped into a butler CHAINED collection to connect all of the various calibrations into a single collection.
+
+.. _collection: https://pipelines.lsst.io/v/daily/modules/lsst.daf.butler/organizing.html
+
+.. _ComCam-Master-Bias-Procedure-Steps:
+
+Procedure Steps
+===============
+
+Once you are logged into LOVE, click on the ``MTQueue`` panel, as shown in the figure below: 
+
+.. figure:: ./_static/love-mtqueue-panel.png
+    :name: MTQueue-love
+
+     Screenshot of LOVE interface with the "MTQueue" pannel.
+
+
+Load the Script
+---------------
+
+After clicking on the ``MTQueue`` panel, search for the script ``maintel/make_comcam_bias.py`` under ``AVAILABLE SCRIPTS`` on the left, as shown in the figure below:
+
+.. figure:: ./_static/love-available-scripts.png
```

Comment:
I think you cropped the image too much. I can't see the "add" button in the pop-up box. Also, can you add an arrow indicating the "add" button?